include/toolchain.h

```diff
@@ -37,6 +37,11 @@
 #include <toolchain/xcc.h>
 #elif defined(__GNUC__) || (defined(_LINKER) && defined(__GCC_LINKER_CMD__))
 #include <toolchain/gcc.h>
+#if defined(__llvm__)
+#include <toolchain/llvm.h>
+#endif
+#elif defined(__llvm__)
```

Comment:
Why the #elif? It seems to do the same thing?

Reply:
They cover 2 different cases, note that `if` is actually nested in another case